### ISSUE TITLE
Delta psi is inconsistent in Leafviz

### ISSUE THREAD

**Issue by hanyizhou123** [jackhump/leafviz#21]
Hi-Thank you for your great work！
I encountered some issues when I tried to visualize my difference analysis using leafviz, and I noticed that the delta psi in the table was not equal to the difference in psi in the figure. When I went to the official website of Leafcutter to view sample images, I also noticed this situation.
As shown in the example image on the official website, in the table below, the delta psi of intron ”a“ is -0.124, while the difference in values in the figure is 0.13-0.65=-0.52. Perhaps I misunderstood the meaning of the values in the figure, but I can't figure it out. I hope for an answer. Thank you very much.

<img width="753" alt="截屏2023-08-29 22 21 07" src="https://github.com/jackhump/leafviz/assets/143521723/4588b220-64e9-462c-91f7-f54c37e2f18d">

**Comment by jackhump** [comment]
Hi! Glad you're enjoying Leafviz. Yes this is an issue I've discussed with the Leafcutter team before. 

The "deltaPSI" column comes from the _effect_sizes.txt output of running leafcutter_differential_splicing. This is the difference between the two group intron usage estimates that comes from the Dirichlet multinomial model. 
These estimates are correlated to, but are not identical to just taking the mean intron usage of each group, which is what Leafviz calculates and displays.

**Comment by osbahrm** [comment]
I got a follow-up question on this:
I observed a huge difference between the values in the _effect_sizes file and those displayed in the leafviz plots. 
In leafviz, my 1/1 genotype group shows a junction with zero inclusion
![Unknown-1](https://github.com/user-attachments/assets/80545f1e-929e-4fe9-99e2-d2fa1a870007)
while the _effect_sizes file shows the exact opposite (0.958). Is there any explanation for this?

chr25:14511856:14517382:clu_58953_-	-3.3721694330303	0.95751838193621	0.130427213391246	-0.827091168544964
